dbfit-java/derby/src/main/java/dbfit/environment/DerbyEnvironment.java

```diff
@@ -55,32 +55,32 @@ public Pattern getParameterPattern() {
 
     public Map<String, DbParameterAccessor> getAllColumns(
             final String tableOrViewName) throws SQLException {
-        String qry = "SELECT COLUMNNAME, COLUMNDATATYPE "
+        String qry = "SELECT COLUMNNAME AS COLUMN_NAME, COLUMNDATATYPE AS TYPE_NAME, "
+                + " ROW_NUMBER() OVER() AS ORDINAL_POSITION, " +
```

Comment:
Hmm, it's not something new but now I see an issue in that query... I wouldn't rely on rows order in absence of an `order by` clause. There is an `COLUMNNUMBER` column which perhaps should help for that..